### ISSUE TITLE
Search by artist on the command-line

### ISSUE THREAD

**Issue by Chinggis6** [trizen/clyrics#6]
I've found the package on AUR. It is cool that it supports `cmus` in daemon mode. I'll definitely try it. I've been `azlyrics` from `python-azlyrics-git` using before discovering `clyrics`.

Only thing, how can specify the artist in the search, for instance:

`Clouds - Heaven Was Blind to My Grief`

to retrieve me lyrics of my favourite metal band :-)

I hope it (would) support(s) non-very-popular genres (unlike `azlyrics`) fetching lyrics from [darklyrics](http://darklyrics.com).

**Comment by trizen** [comment]
Hi,

`clyrics` works by making a query on Google with the name of a given song, which can also include the artist name. The search is restricted only to the websites specified in the plugin files. A user, familiar with regular expressions and a little bit of Perl, can add more plugins for various other lyrics sites, using the general structure found in the provided plugins.

Does this answers your question?

**Comment by Chinggis6** [comment]
Yes, it was informative. I'll try to check the plugin files. Thanks.
